async function renderOpportunity(req, res, next) {
  try {
    const availableOpportunities = await 
  } catch (error) {
    next(error)
  }
}
